tests/integration/test_engine/test_api/test_kvdb/data/test_cases/cases_kvdb_api_non_existent_database_content_dump.yaml

```diff
@@ -0,0 +1,23 @@
+- name: Dump the content of a non-existent database
```

Comment:
Solved https://github.com/wazuh/wazuh-qa/pull/3809/commits/7ed825ab6801750a1b5d010e7154b38a94f36a1c

> The TODO field is required by the methods that load the cfg, metadata, and cases. Right now it is not used within these tests. We may need to use it in the configuration module testing (since the engine is configured via file/args/env_vars we may need to add our own way to set its configuration)